src/einsteinpy/symbolic/predefined/vacuum_solutions.py

```diff
@@ -66,6 +66,44 @@ def Kerr(c=constants.c, sch=symbols("r_s"), a=symbols("a")):
 
 
 def KerrNewman(
-    c=constants.c, G=constants.G, sch=symbols("r_s"), a=symbols("a"), Q=symbols("Q")
+    c=constants.c, G=constants.G, eps0=constants.eps0,sch=symbols("r_s"), a=symbols("a"), Q=symbols("Q")
 ):
-    raise NotImplementedError
+    """
+    Kerr-Newman Metric in Boyer Lindquist coordinates.
+
+    Parameters
+    ----------
+    c : ~sympy.core.basic.Basic or int or float
+        Any value to assign to speed of light. Defaults to ``c``.
+    eps0 : ~sympy.core.basic.Basic or int or float
+        Any value to assign to the electric constant or permittivity of free space. Defaults to ``eps0``.
+
+    sch : ~sympy.core.basic.Basic or int or float
+        Any value to assign to Schwarzschild Radius of the central object. 
+        Defaults to ``r_s``.
+    a : ~sympy.core.basic.Basic or int or float
+        Spin factor of the heavy body. Usually, given by ``J/(Mc)``,
+        where ``J`` is the angular momentum.
+        Defaults to ``a``.
+    Q:  ~sympy.core.basic.Basic or int or float
+
+    """
+    coords = symbols("t r theta phi")
+    t, r, theta, phi = coords
+    Sigma = r ** 2 + (a ** 2 * cos(theta) ** 2)
+    rQsq = (Q**2 G) / (4 np.pi eps0 c**4)
```

Comment:
```suggestion
    rQsq = (Q**2 * G) / (4 * sympy.pi * eps0 * c**4)
```

Reply:
python doesn't work this way. :)

Reply:
I was going to do from sympy import pi (i.e. add pi to the list of import). Then use 'pi' in the equation. Could I change that code inside this pull request?

Reply:
Yes, that's alright.
There's no need to open separate PRs pertaining to the same issue
`from sympy import pi` would also work!